.swiftformat

```diff
@@ -79,5 +79,4 @@
 --xcodeindentation enabled
 --yodaswap always
 
---disable redundantReturn,hoistAwait
---swiftversion 5.7
```

Comment:
Fixing `swiftversion` in the config is useful to avoid unnecessary changes in formatting. Could you please elaborate on why it's needed to remove it?

Reply:
This has moved to a file https://github.com/mapbox/mapbox-search-ios/blob/main/.swift-version which is available for other tools too so upgrades have less noise when changing swift version